api_app/analyzers_manager/observable_analyzers/bgp_ranking.py

```diff
@@ -0,0 +1,102 @@
+import json
+import logging
+
+import requests
+
+from api_app.analyzers_manager import classes
+from api_app.analyzers_manager.exceptions import AnalyzerRunException
+from tests.mock_utils import MockUpResponse, if_mock_connections, patch
+
+logger = logging.getLogger(__name__)
+
+
+class BGPRanking(classes.ObservableAnalyzer):
+    """
+    wrapper for https://github.com/D4-project/BGP-Ranking
+    """
+
+    observable_name: str
+    period: int  # optional
+
+    def update(self) -> bool:
+        pass
+
+    def run(self):
+        logger.info("Running BGP_Ranking")
+        urls = {
+            "getASN": "https://bgpranking-ng.circl.lu/ipasn_history/?ip=",
+            "getASNRank": "https://bgpranking-ng.circl.lu/json/asn",
+            "getASNHistory": "https://bgpranking-ng.circl.lu/json/asn_history",
+        }
```

Comment:
Can we set these as class attributes? They don't need to be calculated at every execution

Reply:
oh yeah, right. lemme fix that.